ferrocene/doc/internal-procedures/src/subtree-pulls.rst

```diff
@@ -35,6 +35,18 @@ request automatically, only requiring sign off of a reviewer to get merged.
 Sometimes merge conflicts may occur in which case automation opens an issue
 describing the conflict and requiring the upstream pull to be done manually.
 
+Backtrace subtree pull
+^^^^^^^^^^^^^^^^^^^^^^
+
+When `rust-lang/backtrace-rs` is updated, the modifications to its `Cargo.toml`
+file must be done in `std/cargo.toml` instead, where a comment marks the start of
+the `backtrace` crate dependencies.
```

Comment:
```suggestion
When `rust-lang/backtrace-rs` is updated, the modifications to its `Cargo.toml`
file must be copied over to the relevant section in `library/std/Cargo.toml` instead,
where a comment marks the start of the `backtrace` crate dependencies.

This is due to `std` including the sources of backtrace-rs directly, instead of
depending on it as a cargo cargo dependency.
```

Reply:
So for clarity, the changes should be in both `cargo.toml` ?